README.md

```diff
@@ -1,139 +1,73 @@
-
-4. Opportunities: mmj2 was begun as an experimental study of Metamath, and development has proceed in stages, with each new enhancement package designed to be as modular and independent from the others as possible. These developmental stages -- from .mm database validation and printing, proof verification, grammatical analysis, "proof assistanting", Text Mode Formula Formatting, etc. -- have advanced the state of&nbsp; the art of Metamath programming by a small amount and provide a foundation for further work. mmj2 is now usable (and almost good enough to throw away and rewrite properly!) But there is still more work, more experimentation and more learning to be done.&nbsp;
-
-mmj2's capabilities are invoked using "RunParm" commands that form a language, albeit without looping or "if" statements. Thus, it is easy to enhance and extend mmj2, even for "one offs": code a new program, a new "RunParm" command or two, and modify the mmj.util.BatchMMJ2.java program, and one can immediately gain programmatic access to a fully parsed, proof-verified Metamath database loaded into memory and ready for use. (Note: the [MMJ2 Service][11] Feature allows your program to call mmj2 or to be called by mmj2.)
-
-5. Shortcomings of mmj2:
-
-
- * It is not a "prover".
- * It does not update Metamath .mm databases.
- * Its grammatical analysis module does minimal checking for grammatical ambiguity (this section is stubbed out -- very difficult, requiring lots of work in special cases because the general case is impossible to prove :).
- * It is a single-user system, not designed for concurrent access throughout (but you can run more than one instance of mmj2's Proof Assistant GUI simultaneously.)
+# README for mmj2
+
+mmj2 is a proof assistant for the Metamath language.
+Metamath is a language that lets you express mathematical axioms and theorems.
+The proof assistant includes a GUI for creating proofs,
+proof verification tools, and grammatical/sytnax analysis.
+
+mmj2 was originally developed by Mel O'Cat 2005-2011.
+This version has been further modified by Mario Carneiro and David A. Wheeler.
+This program is released under the terms of the
+GNU General Public License Version 2 or any later version
+(SPDX license identifier GPL-2.0-or-later); see [LICENSE.TXT][].
+
+## Installation
+
+Refer to [INSTALL.html][] for instructions on how to install mmj2.
```

Comment:
Can this html file be viewed from github? If not, I think we should probably use markdown instead.

Reply:
I agree that INSTALL.html should be markdown.  (Another possibility would be to set up a github pages project website for mmj2 with INSTALL.html in it, and then link to that site.)

Reply:
This INSTALL.html file can certainly be viewed from GitHub. But if you'd prefer it to be in markdown, I can do that too.

GitHub will automatically redisplay markdown in HTML, so from *GitHub* it works either way. However, if you just download things and the original is in markdown, then you'll only see markdown unless we pregenerate. The original was in HTML, which is why I did it this way, but I can convert to markdown if it's still desired.

Still want it in markdown?

Reply:
Markdown is easier to read from source than HTML (in fact I'm pretty sure that was part of the design). And it is not uncommon to have markdown viewers available if people want to read this offline for some reason (there is a markdown viewer built into vscode, for example, and several web sites have live previewers that you can use for quick markdown rendering).

> This INSTALL.html file can certainly be viewed from GitHub.
When I click on the INSTALL.html on the current readme, I am taken to https://github.com/digama0/mmj2/blob/master/INSTALL.html , which is not rendered. It's possible that it is rendered if you go to another link (which may require setting up a github.io site), but the simple relative link does not have the intended effect. And I have no intention of building a website for mmj2, where these HTML docs would fit in.

Reply:
Ok, fixed.